doc/source/installing_daemons.rst

```diff
@@ -1,145 +1,41 @@
 Installing Rucio daemons
-========================
-
 Prerequisites
-~~~~~~~~~~~~~
-
 The Rucio daemons run on Python 2.7 on any Unix-like platform.
 
 Install via pip
-~~~~~~~~~~~~~~~
-
-Heads up: We recommend to use the docker-based install (see next section) as it will configure many things for you automatically. Only use the pip-based install if you have a good reason and know how to configure your webservices manually:
-
-``pip install rucio``
+Heads up: We recommend to use the docker-based install (see next section) as it will configure many things for you automatically. Only use the pip-based install if you have a good reason and know how to configure your web services manually:
 
-This will pull the latest release from `PyPi <https://pypi.python.org/pypi/rucio/>`_. The Rucio server also needs several Python dependencies. These are all listed in the file ``tools/pip-requires`` and will be pulled in as necessary.
+pip install rucio
 
+This will pull the latest release from PyPi. The Rucio server also needs several Python dependencies. These are all listed in the file tools/pip-requires and will be pulled in as necessary. 
 
 Install via Docker
-~~~~~~~~~~~~~~~~~~
```

Comment:
I would suggest to add the link to relevant dockerhub pages instead of this text to not replicate the information twice and decrease the chance of getting out of sync.